versions.props

```diff
@@ -2,21 +2,21 @@ com.fasterxml.jackson.core:jackson-databind = 2.13.3
 com.github.ben-manes.caffeine:caffeine = 3.1.1
 com.google.code.findbugs:jsr305 = 3.0.2
 com.google.guava:guava = 31.1-jre
-com.jcraft:jzlib = 1.1.3
 com.netflix.concurrency-limits:* = 0.2.2
 com.netflix.feign:feign-*= 8.18.0
 com.palantir.conjure.java:* = 6.57.0
+com.palantir.conjure.java:conjure-lib = 6.57.0
```

Comment:
This shouldn't be necessary, but without it some project fails to resolve dependencies:
```
> Task :conjure-java-server-verifier:compileGeneratedJerseyJava FAILED
> Task :conjure-java-client-verifier:compileGeneratedJerseyJava FAILED

FAILURE: Build completed with 2 failures.

1: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':conjure-java-client-verifier:compileGeneratedJerseyJava'.
> Could not resolve all files for configuration ':conjure-java-client-verifier:generatedJerseyCompileClasspath'.
   > Could not find com.palantir.conjure.java:conjure-lib:.
     Required by:
         project :conjure-java-client-verifier

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':conjure-java-server-verifier:compileGeneratedJerseyJava'.
> Could not resolve all files for configuration ':conjure-java-server-verifier:generatedJerseyCompileClasspath'.
   > Could not find com.palantir.conjure.java:conjure-lib:.
     Required by:
         project :conjure-java-server-verifier

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
==============================================================================
```

I think this is the same issue we've seen in our internal audit library, but I don't have the bandwidth to debug it now.